Documentation/CallGuide.md

```diff
@@ -109,60 +89,75 @@ let sessionConstraints = {
 try {
 	const offerDescription = await peerConnection.createOffer( sessionConstraints );
 	await peerConnection.setLocalDescription( offerDescription );
-
-	// Send the offerDescription to the other participant.
 } catch( err ) {
 	// Handle Errors
 };
+```
+
+## Step 4 - Handling the ICE Candidates on both sides
+
+
+
+```javascript
 
-// Send the offer description to the second client.
-// Usually the data is in JSON so very easy to send on.
 ```
 
-## Step 4 - Set the remote description, create an answer, set the local description
+## Step 5 - Set the remote description, create an answer, set the local description
 
 
 
 ```javascript
 try {
-	// Use the received offerDescription
 	let offerDescription = new RTCSessionDescription( offerDescription );
 	await peerConnection.setRemoteDescription( offerDescription );
 
 	const answerDescription = await peerConnection.createAnswer( sessionConstraints );
 	await peerConnection.setLocalDescription( answerDescription );
-
-	// Send the answerDescription back as a response to the offerDescription.
 } catch( err ) {
 	// Handle Errors
 };
 ```
 
-## Step 5 - Set the remote description
+## Step 6 - Set the remote description
 
 
 
 ```javascript
 try {
-	// Use the received answerDescription
 	await peerConnection.setRemoteDescription( answerDescription );
 } catch( err ) {
-	// Handle Errors
+	// Handle Error
 };
 ```
 
-## Step 6 - 
+## Step 7 - 
 
 
 
 ```javascript
 
 ```
 
-## Step 7 - 
+## Step 8 - 
 
 
 
 ```javascript
 
 ```
+
+## Step 9 - 
+
+
+
+```javascript
+
+```
+
+## Step 10 - 
```

Comment:
Drop the empty steps?

Reply:
That entire guide is unfinished and currently going through a rewrite 👍🏻 
Coming very soon.